src/gwt/src/org/rstudio/core/client/command/ShortcutManager.java

```diff
@@ -34,12 +34,15 @@
 import org.rstudio.core.client.events.NativeKeyDownHandler;
 import org.rstudio.studio.client.RStudioGinjector;
 import org.rstudio.studio.client.application.events.EventBus;
+import org.rstudio.studio.client.events.BeginCopyEvent;
```

Comment:
This should be consolidated into `BeginEditEvent`, `EndEditEvent` with enum for the edit type (cut, copy, paste)